src/python_testing/TC_MCORE_FS_1_3.py

```diff
@@ -128,6 +129,7 @@ async def commission_via_commissioner_control(self, controller_node_id: int, dev
 
         await self.send_single_cmd(
             node_id=controller_node_id,
+            endpoint=1,
```

Comment:
We will need to get this value some other way we cannot hardcode this. Likely we need this value to come in as a command line argument